### PR TITLE
Search fix

### LINKED ISSUE #453: Alias
**Describe the bug**
Alias Searching broke from #412

**To Reproduce**
1. Go to HomePage
2. Click on Alias and Filer
3. Broken

### PR DESCRIPTION
**Pull request by marshyonline**
# Description

Fixes Alias Filtering

Fixes #453

## Type of change

Please delete options that are not relevant.

- [X ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Alias Filter works against prod DB

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated changelog.md with changes made

### CODE REVIEW

File: server/routes/api.js [hunk 1]
```diff
@@ -607,7 +607,7 @@ router.route('/messageSearch')
               qb2.select('id').from('capcodes').where('agency', agency).where('ignore', 0);
           })
           if (alias != '')
-            qb.where('alias.id',alias);
+            qb.where('alias_id',alias);
```
Comment (davidmckenzie) [inline server/routes/api.js:610]: should this be `messages.alias_id`? can't remember the data structure here so just going on context of the other `qb.where` things

Comment (marshyonline) [reply server/routes/api.js:610]: alias.id is not the valid column name 
`[pmon]  2021-01-23 20:27:17  error : select `messages`.*, `capcodes`.`alias`, `capcodes`.`agency`, `capcodes`.`icon`, `capcodes`.`color`, `capcodes`.`ignore` from `messages` left join `capcodes` on `capcodes`.`id` = `messages`.`alias_id` where `alias`.`id` = 'RFS HEART BEAT' order by `messages`.`timestamp` desc - SQLITE_ERROR: no such column: alias.id`

Comment (marshyonline) [reply server/routes/api.js:610]: oh sorry, I miss read
let me check

Comment (marshyonline) [reply server/routes/api.js]: Fixed in last commit

**Review by davidmckenzie — APPROVED** [review]

**Pull request merged**